### PR TITLE
Initial_setup

### PR DESCRIPTION
**Pull request by Nuri1977**
## Milestone1: Inital sepup

- Linters added
- Added gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# hello-microverse
+# Hello Microverse
+
+My first project at Microverse.
```
Comment (rotimiazeez) [inline README.md]: - [ ] Kindly Make the readme file more descriptive and professional. You can see example [here](https://github.com/microverseinc/readme-template)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Nuri1977 👋, 

### Good job so far!🔥🔥

#### There are some issues that you still need to work on to go to the next project but you are almost there!🙏🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Nuri1977) [reply README.md]: Thank you @rotimiazeez .

Comment (Nuri1977) [reply README.md]: Hi @rotimiazeez, with the changes I made on the Readme.MD file did I pass the Code review process? If not then please give me instructions on what to do?

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# hello-microverse
+# Hello Microverse
+
+My first project at Microverse. The main goal of this project is to learn the basics of Github workflow, linters and Code Review flow.
```
Comment (leolpaz) [inline README.md:3]: - [ ] It seems you got a bit confused here, I'm gonna leave a screenshot of the requirement below, so kindly use the template from this [repo](https://github.com/microverseinc/readme-template),, to copy it, click on the README.md file in the repo I linked to you, click in RAW here:
image
and then copy the entire template to your README file and fill it with relevant information for this project.

image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Nuri1977 , 

Great work on working on the changes requested by a previous reviewer 👏 
You've done well working on the requested change, but the problem is not addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Nuri1977) [reply README.md:3]: I got confused with the requirements. I did not scroll down and thought that only the Header and paragraph are necessary for the Readme file.

Comment (Nuri1977) [reply README.md:3]: @leolpaz thank you for the explanation with the picture you send me.

Comment (Nuri1977) [reply README.md:3]: @leolpaz please send me additional information on how to finish the Code Review process successfully.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Nuri1977 👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**